internal/mode/static/nginx/modules/src/httpmatches.js

```diff
@@ -44,7 +46,12 @@ function redirect(r) {
     return;
   }
 
-  r.internalRedirect(match.redirectPath);
+  let args = qs.stringify(r.args);
```

Comment:
does this mean we had a bug before?

Reply:
No, args were contained properly within the `request_uri`. However, rewrites don't use `request_uri`, so the args were lost.

Reply:
would it make sense to add a comment there in NJS code  explaining that?